uitools/examples/cpp_quick/src/main.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright 2021 ESRI
+//
+// All rights reserved under the copyright laws of the United States
+// and applicable international laws, treaties, and conventions.
+//
+// You may freely redistribute and use this sample code, with or
+// without modification, provided you include the original copyright
+// notice and use restrictions.
+//
+// See the Sample code usage restrictions document for further information.
+//
+
+#include "Esri/ArcGISRuntime/Toolkit/register.h"
+#include "proxies/ArcGISRuntimeEnvironmentProxy.h"
+#include "proxies/GeoModelProxy.h"
+#include "proxies/MapQuickViewProxy.h"
+#include "proxies/SceneQuickViewProxy.h"
+
+#include <QDir>
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+#include <QQuickStyle>
+#include <QtWebView>
+
+#include <ArcGISRuntimeEnvironment.h>
+#include <Map.h>
+#include <MapQuickView.h>
+#include <Scene.h>
+#include <SceneQuickView.h>
+
+//------------------------------------------------------------------------------
+int main(int argc, char* argv[])
+{
+    QtWebView::initialize();
+    QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+    QGuiApplication app(argc, argv);
+
+    // Use of Esri location services, including basemaps and geocoding, requires
+    // either an ArcGIS identity or an API key. For more information see
+    // https://links.esri.com/arcgis-runtime-security-auth.
+
+    // 1. ArcGIS identity: An ArcGIS named user account that is a member of an
+    // organization in ArcGIS Online or ArcGIS Enterprise.
+
+    // 2. API key: A permanent key that gives your application access to Esri
+    // location services. Create a new API key or access existing API keys from
+    // your ArcGIS for Developers dashboard (https://links.esri.com/arcgis-api-keys).
+
+    const QString apiKey = QString("");
```

Comment:
💭 @AndrewBladon added some code to the samples recently that lets you set an api key via the cmd line. That means you don't have to compile it in which is quite nice